src/components/layout/sendTip/SendTip.vue

```diff
@@ -243,15 +252,61 @@ export default {
         this.sendingPost = false;
       }
     },
+    async uploadImage(event) {
+      this.uploadingMedia = true;
+
+      const formData = new FormData();
+      formData.append('image', event.target.files[0]);
+      fetch('https://api.imgur.com/3/image.json', {
+        method: 'post',
+        body: formData,
+        headers: {
+          Authorization: `Client-ID ${process.env.VUE_APP_IMGUR_API_CLIENT_ID}`,
+        },
+      })
+        .then((res) => res.json())
```

Comment:
Use `async`/`await` instead

Reply:
So, this was fixed later, but it is unclear in per-commit reading. @AtanasKrondev you may not fix it, but you got the idea